website/client/components/challenges/challengeDetail.vue

```diff
@@ -30,7 +30,7 @@
           .details(v-once) {{$t('participantsTitle')}}
         .box
           .svg-icon.gem-icon(v-html="icons.gemIcon")
-          | {{challenge.prize}}
+          challenge.prize(v-if=null) {{challenge.prize = 0}}
```

Comment:
Is this necessary with the other PR + migration going to make it impossible to have challenges with a prize of 0? I'd only add a new check when creating the challenge

Reply:
What do you mean? The reason I did this was because that way the currently null challenge prizes that haven't been changed would show up as 0 gem prize - do you mind elaborating a bit? Thanks heaps! =)

Kira

Reply:
What do you mean? The reason I did this was because that way the currently null challenge prizes that haven't been changed would show up as 0 gem prize - do you mind elaborating a bit? Thanks heaps! =)

Kira

Reply:
Sure! The reason why I don't think this is necessary is that when the other PR will be merged we'll also run a migration to convert all existing null prize challenges to having a prize of 0. 

Also, there's a problem with this code: in vue templates there should only be view logic (for example show a 0 if challenge.prize is null) while in this case it would be modifying the value of challenge.prize. Also the syntax of the v-if is wrong because it checks if the value passed to it is converted to true or false and null is always false.

It should be used like this at the existing like 33

```
          | {{challenge.prize || 0}}
```

which means should challenge.prize or 0 if the value is absent

let me know if anything is not clear :)